pages/linux/zramctl.md

```diff
@@ -15,7 +15,7 @@
 
 `sudo modprobe zram num_devices={{2}}`
 
-- Find and initialise the next free zram device to a 2GB virtual drive using LZ4 compression:
+- Find and initialise the next free zram device to a 2 GB virtual drive using LZ4 compression:
 
 `sudo zramctl --find --size {{2GB}} --algorithm {{lz4}}`
```

Comment:
This page `linux/zramctl.md` uses `zram` in the descriptions — shouldn't it be `ZRAM`?

Reply:
Just peeked now but I think `zram` is fine?

> Initially, zram had only the latter function, hence the original name "compcache" ("compressed cache"). 
> \- https://en.wikipedia.org/wiki/Zram

> The zram module creates RAM-based block devices named /dev/zram<id> (<id> = 0, 1, …).
\- https://www.kernel.org/doc/html/latest/admin-guide/blockdev/zram.html